### ISSUE TITLE
A possible bug in pipeline_ddpm.py with start_t

### ISSUE THREAD

**Issue by ccj5351** [DQiaole/FlowDiffusion_pytorch#4]
Hello,

Thanks for sharing the code. There is a possible bug for your check:

```python
if start_t is None:
            model_input = torch.cat([inputs[:, :6], image], dim=1)
            timesteps = self.scheduler.timesteps
        else:
            model_input = inputs
            timesteps = self.scheduler.timesteps[-start_t:]
            # need your check
            # since we do not want to use a random noise for start_t being
            # a non-zero step (for example: the coarse-to-fine pattern)
            image = inputs[:,-2:,:,:] # use coarse flow you predicted already
```

**Comment by ccj5351** [comment]
This change will help improve the accuracy when running [evaluate_diffusers_tile.py](https://github.com/DQiaole/FlowDiffusion_pytorch/blob/main/evaluate_diffusers_tile.py).

**Comment by DQiaole** [comment]
Thank you for your pull request. You are right, it is better to use coarse flow as the initialization of noised image.
